### DISCUSSION TITLE
Sudden spike in CPU and Disk's usage causing the server to be taken down

### DISCUSSION THREAD

**Discussion by a7maadf** [firefly-iii/firefly-iii#7962] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [X] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

Since updating to v6.0.23, I've been facing an issue with my Docker instance on an Azure VM. Initially, the instance runs smoothly for a few days, but then it suddenly experiences significant spikes in both CPU and Disk usage. This surge causes the server to become unresponsive, making both the web interface and SSH connections inaccessible.

I'm uncertain about the root cause of this problem. To retrieve debug information, I had to reboot the server. Hence, I'm unsure about the relevance or accuracy of the logs at this stage.

I would appreciate any help you could provide. 

Thx <3 

![image](https://github.com/firefly-iii/firefly-iii/assets/89315818/84037815-5fdc-4e85-af69-7c962fc7dcd8)

### Debug information

Debug information generated at 2023-09-14 20:51:04 for Firefly III version **v6.0.23**.

<table><thead><tr><th colspan="2">System information</th></tr><tr><th>Item</th><th>Value</th></tr></thead><tbody><tr><td>Firefly III</td><td>v6.0.23 / v2.0.7 / 20(exp. 20)</td></tr><tr><td>PHP version</td><td>8.2.9 / apache2handler / Linux</td></tr><tr><td>BCscale</td><td>12</td></tr><tr><td>Error reporting</td><td>Display: Off, reporting: ALL errors</td></tr><tr><td>Max upload</td><td>67108864 (64 MB)</td></tr><tr><td>Database drivers</td><td><strong>*mysql*</strong>, pgsql, sqlite, </td></tr><tr><td>Docker build</td><td><span>#</span>636, base <span>#</span>37</td></tr></tbody></table><table><thead><tr><th colspan="2">Firefly III information</th></tr><tr><th>Item</th><th>Value</th></tr></thead><tbody><tr><td>Timezone</td><td>America/Chicago + UTC</td></tr><tr><td>App environment</td><td>local, debug: true</td></tr><tr><td>Logging</td><td>notice, stack / (empty)</td></tr><tr><td>Cache driver</td><td>file</td></tr><tr><td>Default language and locale</td><td>en_US + equal</td></tr><tr><td>Trusted proxies</td><td>**</td></tr><tr><td>Login provider & user guard</td><td>eloquent / web</td></tr><tr><td>Login headers</td><td>N/A + N/A</td></tr><tr><td>Stateful domains</td><td></td></tr></tbody></table><table><thead><tr><th colspan="2">User-specific information</th></tr><tr><th>Item</th><th>Value</th></tr></thead><tbody><tr><td>User</td><td><span>#</span>1 of 1</td></tr><tr><td>User flags</td><td>:pig: :wrench: :email:</td></tr><tr><td>Session start</td><td></td></tr><tr><td>Session end</td><td></td></tr><tr><td>View range</td><td>1M</td></tr><tr><td>User language</td><td>en_US</td></tr><tr><td>User locale</td><td>en_US</td></tr><tr><td>Locale(s) supported</td><td>en_US.utf8: :white_check_mark:<br>en_US.UTF-8: :white_check_mark:<br></td></tr><tr><td>User agent</td><td>Mozilla/5.0 (Windows NT 10.0; rv:109.0) Gecko/20100101 Firefox/117.0</td></tr></tbody></table>

**Comment by JC5** [comment]
Sorry I have nothing to go on. For all I know, some script kiddie is hammering away at the server. Access logs should at least reveal what's happening.
